feedthefox/urls.py

```diff
@@ -12,7 +12,7 @@
     '',
     url(r'^$', 'feedthefox.base.views.home', name='home'),
     url(r'^devices/$', 'feedthefox.devices.views.devices', name='devices'),
-    url(r'^device/$', 'feedthefox.devices.views.device', name='device'),
+    url(r'^device/(?P<id>[0-9]+)$', 'feedthefox.devices.views.device', name='device'),
```

Comment:
Since model and manufacturer should be unique, we can exploit this display here something more meaningful.